AlliCrab/ViewControllers/SettingsTableViewController.swift

```diff
@@ -137,12 +137,16 @@ class SettingsTableViewController: UITableViewController {
         
         switch tableViewSection {
         case let .userScript(userScript):
-            guard let author = userScript.author else {
-                return nil
+            var scriptFooter = userScript.description
+            if userScript.author != nil {
```

Comment:
Would prefer if you used optional binding here instead of the nil check and forced unwrap:
```suggestion
            if let author = userScript.author {
```